### ISSUE TITLE
Improve alignment of buttons and titles

### ISSUE THREAD

**Issue by nedrichards** [flathub/frontend#178]
![Screenshot from 2022-01-31 20-50-38](https://user-images.githubusercontent.com/302677/151871032-65639f85-3084-4ac0-944b-419445b87e26.png)

In the attached screenshot the alignment of the button and title seems quite off, it leads to a less composed presentation for the section.

**Comment by razzeee** [comment]
Yes, saw that too the other day, when I prepared the hide show more button pr

**Comment by razzeee** [comment]
Looking at this, I'm not sure which layout would make more sense.
![image](https://user-images.githubusercontent.com/5943908/152237151-3ea30f68-29ea-44fe-ae9b-d3b507f776bf.png)

I would think everything else would be more weird. Going for a smaller button might help, but is probably not what we want.

We might need to address the case, that the header row will be smaller, when the button is not there, but shouldn't be a problem as long as we don't use them next to each other (tm)

**Comment by nedrichards** [comment]
Right - the larger button is actually helpful! And yes I agree that this is something where it's OK to have two treatments so long as we can be sure that they won't be seen next to each other (which they really shouldn't be in this design currently)

**Comment by razzeee** [comment]
Well mobile kind of does that ![Screenshot_20220323-015822_Firefox_Nightly.png](https://user-images.githubusercontent.com/5943908/159601134-8f91665e-40ea-4e27-aa3a-0d61d4785474.png)
